src/Models/IoTWorkbenchProjectBase.ts

```diff
@@ -311,7 +312,7 @@ export abstract class IoTWorkbenchProjectBase {
         const projectConfigContent =
             await FileUtility.readFile(
                 type, iotworkbenchprojectFilePath, 'utf8') as string;
-        if (projectConfigContent !== '') {
+        if (projectConfigContent && projectConfigContent !== ' ') {

```

Comment:
>(projectConfigContent && projectConfigContent !== ' ') [](start = 11, length = 54)

suggest using string.trim(),  which will be more easier  to read

Reply:
Fixed.